wordpress-mobile/WordPress-Android
# Issue/7800 retain undo cards saved posts list

Fixes #7800 

#7770 Needs to be merged first.

Undo items are retained across configuration changes. Undo items are removed when the application is closed or when the user changes the filter in Reader.

To test:
1. Save several posts for later
2. Go to saved posts list
3. Remove a post from the saved posts
4. Notice collapsed 'undo' view is displayed
5. Rotate your device
6. Notice the collapsed undo view  is displayed
7. Change Reader filter to Followed sites 
8. Change Reader filter back to Saved posts
9. Notice the collapsed undo view is NOT displayed
